### PR TITLE
Fixes failed AD group import

### LINKED ISSUE #1155: AD groups import fails
We met a problem with AD groups mapping and users import process. When import job started, it fails with this error:
image

The proposal is to change type of the field like this:
alter table webapi.batch_job_execution_params alter string_val type VARCHAR;

### PR DESCRIPTION
**Pull request by wivern**
Fixes #1155

### CODE REVIEW

File: src/main/resources/db/migration/postgresql/V2.8.0.20190531181956__alter_job-execution-params_string-val.sql [hunk 1]
```diff
@@ -0,0 +1,121 @@
+DROP VIEW ${ohdsiSchema}.cc_generation;
+DROP VIEW ${ohdsiSchema}.estimation_analysis_generation;
+DROP VIEW ${ohdsiSchema}.pathway_analysis_generation;
+DROP VIEW ${ohdsiSchema}.prediction_analysis_generation;
+DROP VIEW ${ohdsiSchema}.user_import_job_history;
+
+ALTER TABLE ${ohdsiSchema}.batch_job_execution_params ALTER string_val TYPE VARCHAR;
```
Comment (anthonysena) [inline src/main/resources/db/migration/postgresql/V2.8.0.20190531181956__alter_job-execution-params_string-val.sql]: We should not change the `batch_*` tables as these are provided by Spring Batch. It is also unclear why these changes are required here?

Comment (ssuvorov-fls) [reply src/main/resources/db/migration/postgresql/V2.8.0.20190531181956__alter_job-execution-params_string-val.sql]: fixed
